app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -153,13 +172,32 @@ class BrowserToolbarIntegration(
         toolbar.display.setUrlBackground(urlBackground)
     }
 
+    private fun setActionButtons() {
+        toolbar.addNavigationAction(eraseAction)
```

Comment:
Since this action will not get added/removed dynamically, I think we could keep this one in `init()`? 🤔 (Which also avoids us having to remove it in stop() again).